vendor/VREffect.js

```diff
@@ -232,7 +232,7 @@ THREE.VREffect = function ( renderer, onError ) {
 	var cameraR = new THREE.PerspectiveCamera();
 	cameraR.layers.enable( 2 );
 
-	this.render = function ( scene, camera ) {
+	this.render = function ( scene, camera, renderTarget, forceClear ) {

```

Comment:
I agree and the change is both trivial and makes VREffect more consistent with the rest of three. I'll try submiting a PR to threejs as well but I suggest we keep the patched version for now.